client/simple/src/js/head/00_init.js

```diff
@@ -1,20 +1,19 @@
 /* SPDX-License-Identifier: AGPL-3.0-or-later */
-(function (w, d) {
-  'use strict';
-
+((w, d) => {
```

Comment:
I'm not really a fan of using the arrow syntax instead of `function` here at the file declarations. Apparently after that, we can no longer reference the variable inside the method with `this` if I understand it correctly, possibly that could break something?

For reference: https://biomejs.dev/linter/rules/use-arrow-function/